window-purpose-fixes.el

```diff
@@ -55,7 +82,11 @@ spliting logic with `pop-to-buffer'."
       (unless (memq (framep (selected-frame)) '(nil t pc))
         (x-focus-frame (selected-frame)))
       (set-window-hscroll window 0))
-    (advice-add 'edebug-pop-to-buffer :override 'purpose--edebug-pop-to-buffer-advice)))
+
+    (purpose-sync-advice 'edebug-pop-to-buffer :override 'purpose--edebug-pop-to-buffer-advice)
+    (add-hook 'purpose-fix-togglers
+              (purpose-advice-syncer 'edebug-pop-to-buffer :override
+                                     'purpose--edebug-pop-to-buffer-advice))))
```

Comment:
These two toggle advice and add-hook pairs seem repetitive, should be able to DRY them up.